src/components/Helpers/Loading.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { css } from 'emotion';
+
+const styles = {
+  wrap: css`
+    margin: 100px auto 0;
+  `,
+  peace: css`
+    display: inline-block;
+    vertical-align: middle;
+    animation-direction: alternate;
+    animation-iteration-count: infinite;
+    animation-duration: 0.5s;
+    animation-timing-function: cubic-bezier(0, 0, 1, 1);
+    transform-origin: bottom;
+    font-size: 50px;
+    animation-name: rotate;
+    @keyframes rotate {
```

Comment:
Does this actually work? I've done keyframes in emotion according to https://github.com/emotion-js/emotion/blob/master/docs/keyframes.md

```js
const rotate = keyframes`
  //...
`
```

Reply:
Yes, it does work.

![loading](https://user-images.githubusercontent.com/1090713/37485382-7f4159cc-2848-11e8-9a2e-192490fba93a.gif)

Reply:
✌️

Reply:
This PR is amazing